week-2/K-array-methods/exercise2.js

```diff
@@ -7,7 +7,7 @@
 var mentors = ["Daniel", "Irina", "Rares"];
 var students = ["Rukmini", "Abdul", "Austine", "Swathi"];
 
-var everyone; // complete this statement
+var everyone = [mentors] + [students]; // complete this statement
```

Comment:
Concatenation of strings is done with "+", but for arrays such as mentors and students the way to concatenate is to use concat(). For example: [1, 2, 3].concat([4, 5, 6]);